### PR TITLE
Add option to mint multiple tokens

### PR DESCRIPTION
**Pull request by jukemaster69**
-mm or -mint-multiple <number> as one of the options for the command "mint"

### CODE REVIEW

File: js/packages/cli/src/cli-nft.ts [hunk 1]
```diff
@@ -32,10 +32,8 @@ programCommand('mint')
   .option('-um, --use-method <string>', 'Optional: Single, Multiple, or Burn')
   .option('-tum, --total-uses <number>', 'Optional: Allowed Number of Uses')
   .option('-ms, --max-supply <number>', 'Optional: Max Supply')
-  .option(
-    '-nvc, --no-verify-creators',
-    'Optional: Disable Verification of Creators',
-  )
+  .option('-nvc, --no-verify-creators', 'Optional: Disable Verification of Creators',)
+  .option('-mm, --mint-multiple <number>', 'Optional: Mint Multiple Tokens',)
```
Comment (goriunov) [inline js/packages/cli/src/cli-nft.ts:36]: Wondering if this should be just --multiple as mint is already in the name

like 

```
mint --multiple
```

File: js/packages/cli/src/cli-nft.ts [hunk 2]
```diff
@@ -48,7 +46,11 @@ programCommand('mint')
       totalUses,
       maxSupply,
       verifyCreators,
+      MintMultiple,
```
Comment (goriunov) [inline js/packages/cli/src/cli-nft.ts:49]: do not start the var name with capital letters use camel case

File: js/packages/cli/src/cli-nft.ts [hunk 3]
```diff
@@ -66,18 +68,32 @@ programCommand('mint')
...
-      url,
-      true,
-      collectionKey,
-      supply,
-      verifyCreators,
-      structuredUseMethod,
-      receivingWallet,
-    );
-  });
+    if (MintMultiple !== undefined) MintMultiple = 1;
```
Comment (goriunov) [inline js/packages/cli/src/cli-nft.ts:71]: This condition is wired so if MintMultiple is not undefined you want to set it to 1 ? In this case, if will always be 1,,,

you probably want to have just default value on the variable and do not check it here at all

File: js/packages/cli/src/cli-nft.ts [hunk 4]
```diff
@@ -66,18 +68,32 @@ programCommand('mint')
...
+    const NUMBER_OF_NFTS_TO_MINT = parseInt(MintMultiple, 10);
```
Comment (goriunov) [inline js/packages/cli/src/cli-nft.ts:72]: First name convention... second this parse should be done before any validation like (MintMultiple > maxSupply). I believe   the library should parse things for you already (need to double check that)

File: js/packages/cli/src/cli-nft.ts [hunk 5]
```diff
@@ -66,18 +68,32 @@ programCommand('mint')
...
+        structuredUseMethod,
+        receivingWallet,
+        index + 1,
+        );
+        }
+      };
+  
+      await mintNFT(0);
+  
+      if(MintMultiple > 1) log.info('minted ${NUMBER_OF_NFTS_TO_MINT} tokens');
```
Comment (goriunov) [inline js/packages/cli/src/cli-nft.ts:94]: Why do you need this if check... The logs are exactly the same...

File: js/packages/cli/src/cli-nft.ts [hunk 6]
```diff
@@ -66,18 +68,32 @@ programCommand('mint')
...
+        supply,
+        verifyCreators,
+        structuredUseMethod,
+        receivingWallet,
+        index + 1,
+        );
+        }
+      };
+  
+      await mintNFT(0);
```
Comment (goriunov) [inline js/packages/cli/src/cli-nft.ts:92]: I don't think closure is the right way in here. You should use a simple loop..

**Review by goriunov — CHANGES_REQUESTED** [review]

File: js/packages/cli/src/cli-nft.ts (hunk 3 above)
Comment (stegaBOB) [reply js/packages/cli/src/cli-nft.ts:71]: because its changing it it wont even run since its set as a const.

**Pull request closed without merging**